essyncer-saas/.env.sample

```
@@ -1,2 +1,13 @@
-ELASTICSEARCH_URL=http://elasticsearch:9200
\ No newline at end of file
+CORE_MONGO_URL=mongodb://mongo/erxes_core
+
+# Elasticsearch Configuration
+ELASTICSEARCH_URL=http://elasticsearch:9200
+
+# Plugins Configuration
+PLUGINS_JSON_URL=https://your-domain.com/plugins.json
+
+# Performance Configuration
+MaxPoolSize=2
```

dotenv-linter (4.0.0)

[warning] [LowercaseKey] The MaxPoolSize key should be in uppercase

(LowercaseKey)